tests/test_application_form.py

```diff
@@ -101,8 +101,9 @@ def test_form_pages_accessible(self):
                     },
                     route_rel,
                 )
-                assert len(results["violations"]) <= 4
-                assert len(results["violations"]) <= 4 or all(
+                # TODO why is this number 5?
```

Comment:
Violations I think should be reported in axe_reports when you run locally?

Reply:
ah nice - thanks Harry!